plugins/regrass.cpp

```diff
@@ -343,134 +361,81 @@ int regrass_tile(color_ostream &out, const regrass_options &options, df::map_blo
         mat == tiletype_material::GRASS_DARK ||
         mat == tiletype_material::PLANT ||
         mat == tiletype_material::TREE)
-    {   // Already appropriate tile
-        DEBUG(log, out).print("Tiletype no change.\n");
+    {   // Already appropriate tile.
+        TRACE(log, out).print("Tiletype no change.\n");
         return 1;
     }
-    /*else if (mat == tiletype_material::STONE || // DF doesn't seem to remove mud
+    /*else if (mat == tiletype_material::STONE || // Note: DF doesn't seem to remove mud.
```

Comment:
is this dead code? can it be removed?

Reply:
Assuming we don't want to make it an option.